packages/flutter/test/physics/newton_test.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+@TestOn('!chrome') // different numeric behavior.
```

Comment:
this seems very distressing. this code is being used on web, we can't just ignore the differences.

Reply:
Let's change this to `Tags('web_unimplemented')` and treat it as a bug.

Reply:
We have not yet seen differences in how double math is implemented between JS and Dart. We should see identical results here.

Reply:
Agreed we should treat this as a bug. In this case, the rounding is off - something like `8` vs `7.999999999`.